.github/workflows/lint_code.yml

```
@@ -0,0 +1,36 @@
+name: Lint
+
+on:
+  push:
+
+env:
+  TERRAFORM_VERSION: "1.3.6"
+
+jobs:
+  linting:
+    name: Lint Terraform files
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Cancel previous runs
+        uses: styfle/cancel-workflow-action@0.10.0
+        with:
+          access_token: ${{ github.token }}
+
+      - name: Checkout the repository
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ github.head_ref }}
+
+      - name: Install Terraform
+        uses: hashicorp/setup-terraform@v2
+        with:
+          terraform_version: ${{ env.TERRAFORM_VERSION }}
+
+      - name: Run Terraform format
+        run: terraform fmt -recursive -check
+
+      - name: Run tfsec linter
+        id: tfsec
+        uses: aquasecurity/tfsec-action@v1.0.2
```

Same here, this version is outdated
```suggestion
        uses: aquasecurity/tfsec-action@v1.28.0
```

---

Can you please share where did you find new version?

Here latest version is `1.0.2` :thinking: 
- https://github.com/marketplace/actions/tfsec-action
- https://github.com/aquasecurity/tfsec-action

---

`1.28.0` is tfsec version, but GH action doesn't have new version. I think all good here :slightly_smiling_face: